common/lib/capa/capa/inputtypes.py

```diff
@@ -592,8 +592,11 @@ def get_attributes(cls):
                                              #   set state
             Attribute('width', "400"),       # iframe width
             Attribute('height', "300"),      # iframe height
-            Attribute('sop', None)           # SOP will be relaxed only if this
-                                             # attribute is set to false.
+            # Title for the iframe, should be supplied by the author of the problem. Not translated
+            # because default display name is also not translated, and we can't import Django here anyway.
```

Comment:
While I agree that we don't need to translate 'Problem Remote Content', I think the comment about importing Django is a little unnecessarily confusing. Capa strings can be translated without importing Django.